src/views/credits/credits.jsx

```diff
@@ -73,7 +73,7 @@ var Credits = React.createClass({
                     </li>
                     
                     <li>
-                         <img src="//cdn2.scratch.mit.edu/get_image/user/default_170x170.png" alt="Tina Avatar" />
+                         <img src="https://cdn2.scratch.mit.edu/get_image/user/25500116_170x170.png" alt="Tina Avatar" />
```

Comment:
Can you remove the `https:` (see other lines)? Otherwise this looks good!

Reply:
yep ookay